src/uct/api/v2/uct_v2.h

```diff
@@ -527,33 +538,41 @@ typedef struct uct_iface_is_reachable_params {
      * mask will be ignored. Provides ABI compatibility with respect to adding
      * new fields.
      */
-    uint64_t                     field_mask;
+    uint64_t                       field_mask;
 
     /**
      * Device address to check for reachability.
      * This field must not be passed if iface_attr.dev_addr_len == 0.
      */
-    const uct_device_addr_t      *device_addr;
+    const uct_device_addr_t       *device_addr;
 
     /**
      * Interface address to check for reachability.
      * This field must not be passed if iface_attr.iface_addr_len == 0.
      */
-    const uct_iface_addr_t       *iface_addr;
+    const uct_iface_addr_t        *iface_addr;
 
     /**
      * User-provided pointer to a string buffer.
      * The function @ref uct_iface_is_reachable_v2 fills this buffer with a
      * null-terminated information string explaining why the remote address is
      * not reachable if the return value is 0.
      */
-    char                         *info_string;
+    char                          *info_string;
 
     /**
      * The length of the @a info_string is provided in bytes.
      * This value must be specified in conjunction with @a info_string.
      */
-    size_t                        info_string_length;
+    size_t                         info_string_length;
+
+    /**
+     * Reachability scope. Can be one of the following values:
+     * Intra-node: Checks if addresses describe the same device.
+     * Inter-node: Checks reachability between different nodes.
+     * Default is inter-node.
```

Comment:
1. intra_node / inter_node is not a representative name, need something with the word "Device"
2. document these in uct_iface_reachability_scope_t enum

Reply:
see above comment